### PR TITLE
Remove components meta data unused in app

### PR DESCRIPTION
**Pull request by TrySound**
Generated .props.json in react-sdk added huge amount of unused data to app bundle.

Here I loaded only what's used, validated generated data and covered meta with strict types.

As you can see this heavily reduced app size

du -ck public/build
2800
2336

Removed primitive components stories as not very useful to not increase meta size.

## Before requesting a review

- [ ] if the PR is WIP - use draft mode
- [x] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")
- [ ] what kind of review is needed?
  - [x] conceptual
  - [x] detailed
  - [ ] with testing

## Test cases

- [ ] step by step interaction description and what is expected to happen

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

File: packages/react-sdk/bin/generate-arg-types.ts [hunk 1]
```diff
@@ -33,11 +34,22 @@ if (componentFiles.length === 0) {
 // Create a parser with using your typescript config
 const tsConfigParser = withCustomConfig(tsConfigPath, options);
 
+const schema = z.record(
```
Comment (kof) [inline packages/react-sdk/bin/generate-arg-types.ts:37]: convention for zod schemas is always capital letter, ideally call it a proper name, I think this is `ArgTypes`

File: packages/react-sdk/bin/generate-arg-types.ts [hunk 2]
```diff
@@ -33,11 +34,22 @@ if (componentFiles.length === 0) {
...
+    options: z.array(z.string()).optional(),
+  })
+);
+
 // For each component file generate argTypes based on the propTypes
 componentFiles.forEach((filePath) => {
   const jsonPath = filePath.replace(".tsx", ".props.json");
   const res = tsConfigParser.parse(filePath);
   const argTypes = propsToArgTypes(res[0].props);
+  // console.log(argTypes)
```
Comment (kof) [inline packages/react-sdk/bin/generate-arg-types.ts:51]: forgotten

File: packages/react-sdk/src/arg-types/utils.ts [hunk 3]
```diff
@@ -30,77 +29,72 @@ export const propsToArgTypes = (
...
-
-    result[propName] = { ...prop, ...control };
+    const argType = getArgType(prop);
+    if (argType != null) {
+      result[propName] = argType;
+    }
     return result;
-  }, {} as ArgTypes);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  }, {} as Record<string, any>);
```
Comment (kof) [inline packages/react-sdk/src/arg-types/utils.ts:38]: can't you use our zod type here?

File: packages/react-sdk/src/components/meta.ts [hunk 4]
```diff
@@ -1,15 +1,45 @@
...
+import Italic from "./italic.props.json";
+import Superscript from "./superscript.props.json";
+import Subscript from "./subscript.props.json";
+import Link from "./link.props.json";
+import Paragraph from "./paragraph.props.json";
+import Span from "./span.props.json";
+import TextBlock from "./text-block.props.json";
+import Image from "./image.props.json";
+
+const base = {
```
Comment (kof) [inline packages/react-sdk/src/components/meta.ts:17]:
```suggestion
const components = {
```

Comment (kof) [inline packages/react-sdk/src/components/meta.ts:17]:
```suggestion
const meta = {
```

**Review by kof — APPROVED** [review]

File: packages/react-sdk/src/arg-types/utils.ts (hunk 3 above)
Comment (TrySound) [reply packages/react-sdk/src/arg-types/utils.ts:38]: Can't import from bin because of rootDir in tsconfig.json

**Review by istarkov — APPROVED** [review]

**Pull request merged**